content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-Use this page to find the type of solution that best fits your needs.
+Depending on your requirements such as ease of maintenance, security, control, available resources, and level of expertise to operate and manage a Kubernetes cluster there are various solutions available. You can deploy a Kubernetes cluster on a local machine, cloud, on-premise cloud; or choose a managed Kubernetes cluster. You can also create custom solutions across a wide range of cloud providers, or bare metal environments to setup a Kubernetes cluster.
```

Comment:
Some people say "on-premise", but most of the writers I've talked to agree that it should be "on-premises" or "on-prem".

Also, I'm not sure what an on-prem cloud is. It might be fine. Or it might be better to say on-prem datacenter.